test/api_integration/apis/core/index.js

```diff
@@ -34,6 +34,12 @@ export default function ({ getService }) {
           .get('/requestcontext/savedobjectsclient')
           .expect(200, 'SavedObjects client: {"page":1,"per_page":20,"total":0,"saved_objects":[]}')
       ));
+
+      it('provides access to application rendering client', async () => {
+        await supertest.get('/requestcontext/render').expect(200, /app:core/);
+        await supertest.get('/requestcontext/render/core').expect(200, /app:core/);
+        await supertest.get('/requestcontext/render/testbed').expect(200, /app:testbed/);
+      });
```

Comment:
The fact that the associated endpoint is in the testbed plugin bothers me.

Reply:
Could you elaborate on what you mean here?

Reply:
Yeah it might be better if we had a plugin in the api_integration test suite that was loaded to test against similar to how plugin_functional tests. I think we can do that in a separate PR though.